### PR TITLE
Fix retries

### PR DESCRIPTION
**Pull request by jorgecuesta**
- Removed boolean negation that force retries.
- Added session validation before notifying the node.

### CODE REVIEW

File: app/cmd/rpc/mesh/relay.go [hunk 1]
```diff
@@ -137,6 +137,26 @@ func notifyServicer(r *pocketTypes.Relay) {
 		return
 	}
 
+	sessionHash := getSessionHashFromRelay(r)
+
+	if appSession, ok := servicerNode.LoadAppSession(sessionHash); ok {
```
Comment (nodiesBlade) [inline app/cmd/rpc/mesh/relay.go]: `if appSession, ok := servicerNode.LoadAppSession(sessionHash); ok && !appSession.isInvalid`

File: app/cmd/rpc/mesh/relay.go [hunk 2]
```diff
@@ -137,6 +137,26 @@ func notifyServicer(r *pocketTypes.Relay) {
...
+		if !appSession.IsValid {
+			logger.Error(
+				fmt.Sprintf(
+					"relay for app=%s chain=%s servicer=%s was not able to be delivered because session is already invalidated",
+					r.Proof.Token.ApplicationPublicKey,
+					r.Proof.Blockchain,
+					servicerAddress,
+				),
+			)
+			if appSession.Error != nil {
```
Comment (nodiesBlade) [inline app/cmd/rpc/mesh/relay.go]: dont need any of this, just return if invalid (lines 152 to 157)

File: app/cmd/rpc/mesh/relay.go (hunk 1 above)
Comment (jorgecuesta) [reply app/cmd/rpc/mesh/relay.go]: fixed

File: app/cmd/rpc/mesh/relay.go (hunk 2 above)
Comment (jorgecuesta) [reply app/cmd/rpc/mesh/relay.go]: fixed

**Review by nodiesBlade — APPROVED** [review]

**Pull request merged**